includes/wikia/controllers/HeliosHelperController.class.php

```diff
@@ -71,14 +85,15 @@ public function updateAntiSpoof() {
 	public function sendConfirmationEmail() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 8, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)